nixos/modules/programs/qgroundcontrol.nix

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Remember to add your module to the module list, else it won't be picked up

Reply:
Whoops, I always forget that part lol